user-guide/Advanced_Functionality/DataMiner_Agents/Installing_a_DMA/Installing_DM_using_the_DM_Installer.md

```diff
@@ -30,7 +30,7 @@ The DataMiner installer allows you to run a default DataMiner installation, whic
 
 1. Make sure you are logged into Windows with the Administrator account. (Do not use a regular user account with administrative rights.)
 
-1. Double-click *Setup.exe*.
+1. Double-click *Setup_20220622092235919.exe*.
```

Comment:
This seems very specific, and I worry that we'll have to update the procedure whenever there's a small fix to the installer so that this version number changes. Could we change it to this less specific alternative?

```suggestion
1. Double-click the setup executable.
```